### DISCUSSION TITLE
Define `devHandler` as `handler`

### DISCUSSION THREAD

**Discussion by Barbapapazes** [nitrojs/nitro#2684] [General]
### Describe the feature

Hello,

Actually, it's possible to define `devHandler` in the configuration file but it uses a direct `defineEventHandler`. This behavior differs from 'handler` and makes it really complicated to use within a module or from a larger config files likes the Nuxt one.

Do you think it could be possible to define a `devHandler` using a simple string that refer to an file?

### Additional information

- [ ] Would you be willing to help implement this feature?

**Comment by pi0** [comment]
`defineEventHandler` is a type helper. `dev` refers to the runtime it runs. As for modules i would suggest to avoid dev handlers also as much as you can unless you really need to run something in builder context.
